analytics_data_api/v0/models.py

```diff
@@ -249,9 +249,19 @@ def get_last_updated(cls):
 class RosterEntry(DocType):
 
     course_id = String()
+    user_id = Integer()
```

Comment:
@dsjen The `user_id` is a numeric unique ID, used in the mysql databases as the user primary key.  We added it because our client requested it, but edX conventions (to which I can no longer find the link!) recommend using `username` as the identifier instead of `user_id`.